src/routes/safe/components/Transactions/TxList/TxSingularDetails.tsx

```diff
@@ -147,7 +141,7 @@ const TxSingularDetails = (): ReactElement => {
   }
 
   const { transaction, txLocation } = indexedTx
-  const TxList = isTxQueued(transaction.txStatus) ? QueueTxList : HistoryTxList
+  const TxList = isTxQueued(transaction.txStatus) && !isPending ? QueueTxList : HistoryTxList
```

Comment:
Why are you checking the pending status? A pending transaction is still thereoterically queued.

Reply:
When it's pending and is in queue, the queue says "Nonce X needs to be executed first" where X is the nonce of the deep-linked tx. It's not useful, so I switched it to history. It also triggered an update when the history is fetched.

Reply:
That's pretty hack, but if it works... Can you add a comment please.